pkg/status/status.go

```diff
@@ -250,7 +250,7 @@ func (s *status) updateStatus(previousStatus *configv1.ClusterOperatorStatus) er
 		// Log Conditions
 		log.Infof("[status] Attempting to set the ClusterOperator status conditions to:")
 		for _, statusCondition := range s.clusterOperator.Status.Conditions {
-			log.Infof("[status] ConditionType: %v ConditionStatus: ConditionMessage: %v", statusCondition.Type, statusCondition.Status, statusCondition.Message)
+			log.Infof("[status] ConditionType: %v ConditionStatus: %v ConditionMessage: %v", statusCondition.Type, statusCondition.Status, statusCondition.Message)
```

Comment:
Please call out that you are fixing a bug in your commit message if this is meant to be included in your PR.

Reply:
my bad I fixed this in passing because I was getting errors and forgot to remove it. Can put this in a separate commit

Reply:
This should be in a separate PR.

Reply:
+1. Moreover I fixed this in #205 since I was changing code there.